libsolidity/ast/Types.h

```diff
@@ -100,16 +100,21 @@ class MemberList
 public:
 	struct Member
 	{
-		Member(std::string _name, Type const* _type, Declaration const* _declaration = nullptr):
+		Member(std::string _name, Type const* _type, Declaration const* _declaration):
 			name(std::move(_name)),
 			type(_type),
 			declaration(_declaration)
 		{
 		}
 
+		static Member native(std::string _name, Type const* _type)
```

Comment:
The name is a bit confusing. Do we really need this? Can't we just add `nullptr` to the end instead?

Reply:
If I understand correctly this stand for EVM opcodes? We call them "builtins" most of the time.

Reply:
How is this different from `Member::function` though?

Reply:
@hrkrshnn actually, I'm fine either way. I just wanted to shorten some source locations and native(...) just makes sense here (because natives don't have a declaration in the AST).

`s/native/builtin/g` (I'm going to adapt  that).